ebos/eclmpiserializer.hh

```diff
@@ -27,7 +27,7 @@ namespace Opm {
 
 class EclMpiSerializer {
 public:
-    EclMpiSerializer(Dune::MPIHelper::MPICommunicator comm) :
+    EclMpiSerializer(Dune::CollectiveCommunication<Dune::MPIHelper::MPICommunicator> comm) :
```

Comment:
Is there a reason that this constructor is not marked `explicit`?

Reply:
no, it should be. i'll include that in upcoming PR that depend on this, so i don't have to rebase this one.